.eslintrc.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
                 }
             },
             rules: {
+                "react/jsx-no-target-blank": "error",
```

Comment:
I was surprised this wasn't already on. I had mistakenly thought that this was included with the recommended preset.